include/odp/api/spec/time.h

```diff
@@ -65,6 +65,36 @@ extern "C" {
  */
 odp_time_t odp_time_local(void);
 
+/**
+ * Current local time in nanoseconds
+ *
+ * Like odp_time_local(), but the time stamp value is converted into nanoseconds.
+ *
+ * @return Local time stamp in nanoseconds
+ */
+uint64_t odp_time_local_ns(void);
+
+/**
+ * Current local time (strict)
+ *
+ * Like odp_time_local(), but reads the time stamp value more strictly in the program order.
+ * The function may decrease CPU performance around the call, as it may include additional
+ * barrier instructions or otherwise limit out-of-order execution.
+ *
+ * @return Local time stamp
+ */
+odp_time_t odp_time_local_strict(void);
+
+/**
+ * Current local time in nanoseconds (strict)
+ *
+ * Like odp_time_local_ns(), but reads the time stamp value more strictly (see
+ * odp_time_local_strict() documentation) in the program order.
```

Comment:
Or: `Like odp_time_local_strict(), but the time stamp value is converted into nanoseconds.`

Reply:
Fixed in v3.